assets/models/system/Mixtral-8x7B-v0-1/spec.yaml

```diff
@@ -0,0 +1,27 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: mistralai-Mixtral-8x7B-v01
+path: ./
+properties:
+  inference-recommended-sku: Standard_NC24s_v3, Standard_NC24rs_v3,
+    Standard_ND40rs_v2, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4, Standard_ND96amsr_A100_v4, Standard_ND96asr_v4    
+  languages: EN
+tags:
+  Featured: ''
+  SharedComputeCapacityEnabled: ''
+  inference_compute_allow_list:
```

Comment:
Please add the validations performed in the description. Essentially below details should be furnsihed: 
1. if save to blobstorage component was used, please add a run to that
2. Inference endpoint if active or a sample req response snapshotted
3. Successful batch validation run
4. MLmodel and conda.yaml

Reply:
sample PR: https://github.com/Azure/azureml-assets/pull/1917

Reply:
added endpoint and screenshot

Mixtral model cannot yet be loaded with huggingface so batch scenario may not work.
Also model weights are currently in .pt files and works with vllm only (supported in the foundation-model-inference container)

Reply:
btw I see in other models there's SHA property, how are they generated?

Reply:
in this case it will be git commit hash for model we have clones from OSS repo

Reply:
I tried matching config.json of base commit with the registered model. Except   `"sliding_window": 4096,` everything matched with [commit: 5802963a855a4bbe1c7c0bad462759cc4c0cb1d5#d2h-025836](https://huggingface.co/mistralai/Mixtral-8x7B-v0.1/commit/5802963a855a4bbe1c7c0bad462759cc4c0cb1d5#d2h-025836). Was this manually added?

Reply:
Yeah this I believe is what Gaurav got it from Mistral link directly not from HF before HF released it

Reply:
Thanks Miseon